tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -11976,6 +12141,26 @@ make_type_from_opc(ILI_OP opc)
   case IL_DABS:
     llt = make_lltype_from_dtype(DT_DBLE);
     break;
+#ifdef TARGET_SUPPORTS_QUADFP
+  case IL_RQUAD:
+  case IL_DQUAD:
+  case IL_QFLOATK:
+  case IL_QFLOAT:
+  case IL_QCJMP:
+  case IL_QCJMPZ:
+    llt = make_lltype_from_dtype(DT_QUAD);
```

Comment:
It seems a bit redundant here, maybe you can merge these cases which make `lltype` from `DT_QUAD`.

Reply:
Fixed